### ISSUE TITLE
Network inference on ZCU102 but unable to resolve WARNING during Vitis AI compilation (hard-sigmoid cannot by transformed to hard-sigmoid-fix)

### ISSUE THREAD

**Issue by SeanMarotta** [Xilinx/Vitis-AI#1174]
Hi everyone,

My goal is to infer an **neural network on the ZCU102**. I have mastered the different stages of development, from the floating h5 model to the application on the ZCU102. But I encounter a warning during the compilation phase that I am unable to solve. It is an **operation not supported by the DPU**, here's the error :

![warning_compilation](https://user-images.githubusercontent.com/36741611/219089886-380634db-2f3c-4fa0-ae2b-be32ceb4d9b0.png)

This operation appears at other layers of the network and is fortunately supported by the DPU.
Can someone explain to me why this warning is appearing as it is preventing me from running the application on the ZCU102?

For your information, the neural network is an **EfficientNet-B0 from TensorFlow** trained with our **own dataset**.
I have read the Xilinx Documentation (latest ones) :

- _PG338_ to create my own DPU with specific configurations on the Vivado Flow
- _UG1144_ to create my Linux Image for all the necessary drivers and libraries
- _PG1414_ in order to understand the different development phases of Vitis-AI and to adapt them to my needs
- _UG1354_ to create my own application that allows me to do classification using the resources of the PL side

I'm afraid I've skipped over a step that leads me to this **problem** so I am counting on you and your help to help me **solve it**,

Thank you in advance
